src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1321,7 +1321,7 @@ namespace Google.Example.Library.V1.Snippets
             // Create client
             LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
             // Initialize request argument(s)
-            IResourceName name = new ShelfName("[SHELF_ID]");
+            IResourceName name = new ArchivedBookName("[ARCHIVE_PATH]", "[BOOK_ID]");
```

Comment:
library_gapic.yaml specifies the `ListStringsResponse` to be an `AnyResourceNameConfig` via the `*` value, so it's ok that the instantiating type for the `name` parameter changes.